src/pages/Cart/Cart.js

```diff
@@ -0,0 +1,152 @@
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import './Cart.scss';
+
+const Cart = () => {
+  const navigate = useNavigate();
+  const [cartItems, setCartItems] = useState([]);
+  const [selected, setSelected] = useState([]);
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/data.json')
+      .then(res => res.json())
+      .then(data => setCartItems(data));
+  }, []);
+
+  const handleCheck = (checked, id) => {
+    if (checked) {
+      setSelected(prev => [...prev, id]);
+    } else {
+      setSelected(selected.filter(ele => ele !== id));
+    }
+  };
+
+  const handleAllCheck = () => {
+    if (isAllChecked) {
+      const idArr = [];
+      cartItems.forEach(ele => idArr.push(ele.id));
+      setSelected(idArr);
+    } else setSelected([]);
+  };
+
+  const handleMinus = id => {
+    const tmp = cartItems.filter(item => item.id === id)[0];
+    if (tmp.count <= 1) return;
+    else tmp.count--;
+    const next = [];
+    cartItems.map(item => {
+      if (item.id === tmp.id) next.push(tmp);
+      else next.push(item);
+    });
+    setCartItems([...next]);
```

Comment:
- 특정 조건을 만족하는 요소를 찾으시려는 거라면, filter 메서드 대신 find 메서드 사용하시면 됩니다.
- map 메서드 하나만 사용해서 더 간결히 해볼 수 있을 것 같습니다.
```suggestion
    const newCartItems = cartItems.map(item => {
      if (item.id === id && item.count > 1) {
        return {
          ...item, count: item.count - 1
        }
      } else return item
    });
    
    setCartItems(newCartItems);
```